docs/fips.md

```diff
@@ -0,0 +1,7 @@
+# FIPS compliance
+
+FIPS (Federal Information Processing Standard) is the US government computer security standard for cryptography modules that include both hardware and software components. Percona Server for MySQL supports FIPS certified module for OpenSSL, enabling US organizations to introduce FIPS-compliant encryption and thus meet the requirements towards data security.  
```

Comment:
Percona Server for MySQL can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a compliant version of OpenSSL. When running in FIPS mode, it applies the standards to all cryptographic operations performed by the server.

Reply:
Updated